platform/mbed_lib.json

```diff
@@ -110,6 +126,71 @@
         },
         "EFR32": {
             "stdio-baud-rate": 115200
+        },
+        "DISCO_L475VG_IOT01A": {
+            "crash-capture-enabled": true,
+            "reboot-crash-report-enabled": true,
+            "fatal-error-auto-reboot-enabled": true
```

Comment:
The reboot is a pretty big default behaviour functional change. Are we sure on that one?

Reply:
Yeah its a big behavior change. But since the default config is to reboot only once(error_reboot_max = 1) it will reboot and halt, that way it shouldn't impact any tests or other tools. Do you still think we should keep the auto-reboot disabled, any issues you are seeing?

Reply:
I'm not seeing any particular issues myself, just seems like it might be a bit unexpected to end users.

Reply:
I see that, I would leave that as it is for now unless I get a request to change it explicitly, hope that's ok.

Reply:
Can we make sure that gets into the release notes, talk to Mohit.